apache/trafficcontrol
# ops and above (admin) should not have any server fields masked

#### What does this PR do?

Since ops/admins can update server objects, no fields should be "masked" or rather converted to **** because subsequent updates (PUT) will result in *** being inserted into the database

#### Which TC components are affected by this PR?


#### What is the best way to verify this PR?

1. Login as an ops user in TP, edit a server, ensure that ilopassword field is not ****. 
2. update the server object and check in the database to ensure the ilopassword is not set to ***

#### Check all that apply
